src/drem/modelling/res_build_stock_postcode.py

```diff
@@ -65,6 +71,22 @@ def _merge_ber_sa(
     return left.merge(right, left_on=left_on, right_on=right_on, **kwargs)
 
 
+@task
+def _merge_cso_ed(
+    left: pd.DataFrame, right: pd.DataFrame, left_on: str, right_on: str, **kwargs,
+) -> pd.DataFrame:
+
+    return left.merge(right, left_on=left_on, right_on=right_on, **kwargs)
+
+
+@task
+def _extract_cso_columns(
+    df: pd.DataFrame, ed: str, res: str, geometry: str,
+) -> pd.DataFrame:
+
+    return df[[ed, res, geometry]]
```

Comment:
Be careful with this one!  df[[cols]] returns a view whereas df.loc[:, [cols]] actually accesses the data, you may get `pandas` bugs down the line warning you about making changes on a view

https://pandas.pydata.org/docs/user_guide/indexing.html